src/styles/main.scss

```diff
@@ -1,3 +1,42 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  --blue: #00acdc;
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
```

Comment:
let's use sass variables in scss instead of css

Reply:
Not fixed